src/realtime/client.ts

```diff
@@ -35,6 +35,7 @@ export class RealtimeSession {
     let socketImplementation: ISocketWrapper | null = null;
     if (typeof window !== 'undefined') {
       socketImplementation = new webWrapper.WebSocketWrapper();
+      // @ts-ignore
```

Comment:
is this because TypeScript isn't aware chrome could be a global? I'm thinking we should add it to globals, perhaps as described here https://stackoverflow.com/a/40826789/1582783

Reply:
Yeah, I assumed it was an issue with my local `tsc`, so I just added these to quiet it, but I'll try what you linked